collections/_programs/gsoc_2020.html

```diff
@@ -11,7 +11,7 @@
 
 <div class="cover" style="z-index: 20">
   <h3 class="black-text title">
-    Google Summer of Code 2020 and Layer5 Community
+    Google Summer of Code (2020 & 2019) and Layer5 Community
```

Comment:
```suggestion
    Google Summer of Code and Layer5 Community
```
As Layer5 will take part in more GSoC, best to not add years.

Reply:
Agreed

Reply:
Done